assets/js/util/react-features.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
If you're cool with this approach, @aaemnnosttv, I can update existing parts of the codebase to use this module. I think we're much better off importing these things from one place where we control the import path than the approach used elsewhere, eg:

https://github.com/google/site-kit-wp/blob/767c01942c2909e69d1a4b03504ab3d270f22552/assets/js/googlesitekit-module.js#L42-L50

Reply:
Yeah, good idea! I definitely prefer a single place to import from 👍